documentation/staging/content/faq/resource-settings.md

```diff
@@ -23,13 +23,63 @@ description: "Tune container memory and CPU usage by configuring Kubernetes reso
 
 #### Introduction
 
-The operator creates a container in its own Pod for each WebLogic Server instance. You can tune container memory and CPU usage by configuring Kubernetes resource requests and limits, and you can tune a WebLogic JVM heap usage using the `USER_MEM_ARGS` environment variable in your Domain YAML file. A resource request sets the minimum amount of a resource that a container requires. A resource limit is the maximum amount of a resource a container is given and prevents a container from using more than its share of a resource. Additionally, resource requests and limits determine a Pod's quality of service.
+The CPU and memory requests and limits for WebLogic server pods usually need to be tuned
```

Comment:
WebLogic server pods -> WebLogic Server Pods (Server is always capitalized in WebLogic Server, and Pods is capitalized elsewhere in the FAQ)

Reply:
Fixed.